src/main/java/pl/marcin/project/runners/WebAppRunner.java

```diff
@@ -0,0 +1,16 @@
+package pl.marcin.project.runners;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import pl.marcin.project.ProjectApplication;
+import pl.marcin.project.entityService.CustomerEntityService;
+
+public class WebAppRunner implements AppRunner {
+
+    @Override
+    public void runApplication() {
+        ConfigurableApplicationContext context = SpringApplication.run(ProjectApplication.class);
+        var service = context.getBean(CustomerEntityService.class);
```

Comment:
🎉  yes, that was the idea!
Can you now use the service and check if works?
I know you have controllers now, but let's for now pretend we don't have them and can just use `beans` like here :)